src/lambdas/initiateDataRequest/validateZendeskRequest.ts

```diff
@@ -213,7 +213,7 @@ const getTodayUtc = (): number => {
 
 const dataPathFormatCorrect = (dataPath: string): boolean => {
   return (
-    /(\w+(\[\d+\])*\.)+\w+(\[\d+\])*[^,]/.test(dataPath) ||
+    /\w+(\[\d+\])*\.\b\w+(\[\d+\])*\b/.test(dataPath) ||
```

Comment:
Dont think Sonar complains about this regex?

Reply:
Hey Richa, it does, it gives the same backtracking message as it does for the other regex that was changed!
<img width="844" alt="Screenshot 2024-03-08 at 11 00 36" src="https://github.com/govuk-one-login/txma-ticf-integration/assets/130994595/4a4fd540-61bc-46f1-812c-5676b84b5079">